website/source/docs/http/sys-replication-secondary.html.md

```diff
@@ -0,0 +1,291 @@
+                "token": "fb79b9d3-d94e-9eb6-4919-c559311133d6",
+                "ttl": 300,
+                "creation_time": "2016-09-28T14:41:00.56961496-04:00",
+                "wrapped_accessor": ""
+        }
+    }
+    ```
+
+  </dd>
+</dl>
+
+# /sys/replication/primary/revoke-secondary
+
+## POST
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Revoke a secondary’s ability to connect to the primary cluster; the secondary will
+	immediately be disconnected and will not be allowed to connect again unless given 
+	a new activation token.
+
+  </dd>
+
+  <dt>Method</dt>
+  <dd></dd>
+
+  <dt>URL</dt>
+  <dd>`/sys/replication/secondary/revoke-secondary(/<id>)`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    <ul>
+      <li>
+        <span class="param">id</span>
+        <span class="param-flags">required</span>
+		The identifier used when fetching the secondary token.
+      </li>
+      <li>
+        <span class="param">lease_id</span>
+        <span class="param-flags">required</span>
+        The ID of the lease to extend. This can be specified as part of the URL
+        or in the request body.
+      </li>
+    </ul>
+  </dd>
+
+  <dt>Returns</dt>
+ <dd>`204` response code.
+ </dd>
+</dl>
+
+# /sys/replication/secondary/enable
+
+## POST
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Enables replication on a secondary using a secondary activation token.
+
+	Caution: this will immediately clear all data in the current cluster!
+
+  </dd>
+
+  <dt>Method</dt>
+  <dd></dd>
+
+  <dt>URL</dt>
+  <dd>`/sys/replication/secondary/enable(/<token>)`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    <ul>
+      <li>
+        <span class="param">token</span>
+        <span class="param-flags">required</span>
+		The secondary activation token fetched from the primary.
+      </li>
+      <li>
+        <span class="param">primary_api_addr</span>
+        <span class="param-flags">optional</span>
+         Set this to the API address (normal Vault address) to override 
+		 the value embedded in the token. This can be useful if the 
+		 primary’s redirect addressis not accessible directly from this cluste
+      </li>
+	  <li>
+      <span class="param">ca_file</span>
+      <span class="param-flags">optional</span>
+       The path to a CA root file (PEM format) that the secondary can use
+	   when unwrapping the token from the primary. If this and ca_path are
+	   not given, defaults to system CA roots.
+      </li>
+	   <li>
+      <span class="param">ca_path</span>
+      <span class="param-flags">optional</span>
+      The path to a CA root directory containing PEM-format files that the
+	  secondary can use when unwrapping the token from the primary. If this
+	  and ca_file are not given, defaults to system CA roots.
+      </li>
+    </ul>
+  </dd>
+
+  <dt>Returns</dt>
+ <dd>`204` response code.
+ </dd>
+</dl>
+
+
+# /sys/replication/secondary/promote
+
+## POST
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Promotes the cluster to become the new primary
```

Comment:
I would add the same note about there only being one primary, and that the other primary should be demoted or disabled first.